# Time of obstacle observation, coordinate frame ID
Header header

# The center point of the cone
sb_geom_msgs/Point2D center

# The radius of the cone (in meters)
float64 radius
